### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by nguyentvan7**

### CODE REVIEW

**Comment by frzyc** [comment]
MOAR!
For example:

```
## Describe your changes
<!--- Provide a general summary of your changes -->

## Issue or discord link
<!--- link relevant issues to this PR, or provide a link to a discord message/thread -->

## Testing/validation
<!--- Add screenshots if possible -->

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)
- [ ] I have commented my code, in hard-to understand areas.
- [ ] I have made corresponding changes to README or wiki.
- [ ] For front-end changes, I have updated the corresponding English translations.
- [ ] Ran `yarn run mini-ci` locally to validate format + lint.
- [ ] If there were format issues, I ran `nx format write` to resolve them automatically.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Describe your changes
+<!--- Provide a general summary of your changes -->
+
+## Issue or discord link
+<!--- link relevant issues to this PR, or provide a link to a discord message/thread -->
+*
```
Comment (frzyc) [inline .github/pull_request_template.md:6]: extra `*`

**Review by frzyc — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (frzyc) [reply .github/pull_request_template.md:6]: or is this intentional?

Comment (nguyentvan7) [reply .github/pull_request_template.md:6]: It's intentional. Issues referenced in a bullet list will have the title automatically inserted, which is convenient

**Review by frzyc — APPROVED** [review]

**Review by frzyc — APPROVED** [review]

**Pull request merged**